tests/test_main.py

```
@@ -309,13 +447,32 @@ def test_main_meshtastic_connection_failure(
+                "mmrelay.main.asyncio.get_running_loop",
+                side_effect=_patched_get_running_loop,
+            ),
+            patch(
+                "mmrelay.main.asyncio.to_thread",
+                side_effect=_inline_to_thread,
+            ),
+        ):
+            try:
+                asyncio.run(main(self.mock_config))
+            except (SystemExit, Exception):
+                pass  # Expected due to connection failures
```

Ruff (0.14.13)

`try`-`except`-`pass` detected, consider logging the exception

(S110)